### ISSUE TITLE
Keeping https://json-ld.org/spec/ updated

### ISSUE THREAD

**Issue by BigBlueHat** [json-ld/json-ld.org#671]
Currently the "Latest" links on https://json-ld.org/spec/ (and in the header menu) are incorrect--as they point to the last draft from the CG and not the current in-progress WG specs.

Consequently, I'd like to fix that. 😄 

@gkellogg I've noticed that the past pattern seems to be to persist the "published" copies of specs to this repo and those copies (as copies) contain their "this version" URLs pointing to their actual published location (sadly lacking all semblance of linked data...or even basic link relationships......which I'd also love to change/fix/improve 😜).

Should we continue the above pattern with the current Working Drafts from the WG? or (more simply) just change the "Latest" links to point to the W3C URLs?

Right now, I'm thinking just change the "latest" URLs...but figured asking would be best. 😄

**Comment by BigBlueHat** [comment]
@gkellogg I like the "leading note" approach stating that things have moved. However, I've not been able to get one (without CSS or JS) to appear above the version links. Respec doesn't seem to have that capability natively, and the note on RDF Semantics seems to have been added by hand (judging by all the inline styles).

I can get a note added above the abstract...but I'd prefer something higher up...

Thoughts?

**Comment by gkellogg** [comment]
It seems that the `annoying-warning` class was added just for this purpose. From https://www.w3.org/StyleSheets/TR/2016/base.css you can see that `annoying-warning` is intended for this:

```css
 * Special Sections
 *   - .note       for informative notes             (div, p, span, aside, details)
 *   - .example    for informative examples          (div, p, pre, span)
 *   - .issue      for issues                        (div, p, span)
 *   - .advisement for loud normative statements     (div, p, strong)
 *   - .annoying-warning for spec obsoletion notices (div, aside, details)
```

Given that it comes up as a banner that does not scroll, that would seem to be adequate.

W3C seem to automatically update obsolete TRs, such as [RDF Concepts]() with code such as the following just above "This Version":

```html
<div id="rdf_1_1_notice" style="border: solid black 1px; padding: 0.5em; background: #FFB;">
  <p style="margin-top: 0; font-weight: bold;">New Version
        Available: "RDF 1.1 Concepts and Abstract Syntax"<span style="padding-left: 2em;"></span>
        (Document Status Update, 25 February 2014)</p>

  <p style="margin-bottom: 0;">The RDF Working Group has produced
  a W3C Recommendation for a new version of RDF which adds
  features to this 2004 version, while remaining compatible.
  Please see <a href="http://www.w3.org/TR/rdf11-concepts">"RDF 1.1
  Concepts and Abstract Syntax"</a> for a new version of this document, and the
  <a href="http://www.w3.org/TR/rdf11-new">"What's New in RDF 1.1"</a> document for the differences between this 
  version of RDF and RDF 1.1.</p>
</div>
```

This may be automatically done, but it rewrite the static HTML, not the ReSpec source, as you say.

Unless you want to get into ReSpec hacking, what we have in place will probably need to do.

**Comment by BigBlueHat** [comment]
What you added in 0f4546484b6394837f19f887b08f5b40d316a954 looks great!

Thanks, @gkellogg!
